core/shared/src/main/scala/org/typelevel/idna4s/core/bootstring/Delimiter.scala

```diff
@@ -21,89 +21,188 @@
 
 package org.typelevel.idna4s.core.bootstring
 
-import cats.data._
+import cats.Show
+import cats.kernel._
 import cats.syntax.all._
+import org.typelevel.idna4s.core._
 
-sealed abstract class Delimiter extends Product with Serializable {
-  def codePoint: Int
+/**
+ * An Unicode code point which acts as a delimiter between the basic code points and the
```

Comment:
At least based on how I think "Unicode" is pronounced 😂 
```suggestion
 * A Unicode code point which acts as a delimiter between the basic code points and the
```